.travis.yml

```diff
@@ -0,0 +1,41 @@
+# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
+
+language: R
+warnings_are_errors: false
+
+matrix:
+  include:
+    - os: linux
+      cache: packages
+      dist: precise
+      sudo: false
+      r: release
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-4.9
+            - g++-4.9
+            - iodbc
+            - libiodbc2
+            - libiodbc2-dev
+      before_install: |
+        mkdir ~/.R
+        cat <<EOF > ~/.R/Makevars
+        CXX1X=g++-4.9
+        CXX1XSTD=-std=c++11
+
+    - os: osx
+      osx_image: xcode8.2
+      sudo: required
+      r: release
+      before_install:
+        - brew update
+      install:
+        - sudo chown -R $(whoami) /usr/local/share/man/man7
```

Comment:
The whoami command fixes a bug that prevents homebrew from linking to the folder where RODBC header files install. I tried many solutions to fix this issue and eventually had to resort to homebrew and this permissions command